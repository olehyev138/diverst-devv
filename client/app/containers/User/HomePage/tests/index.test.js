/**
 *
 * Tests for HomePage
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowWithIntl, loadTranslation } from 'enzyme-react-intl';
import { HomePage } from '../index';
import { intl } from 'tests/mocks/react-intl';

loadTranslation('./app/translations/en.json');
describe('<HomePage />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const wrapper = shallowWithIntl(<HomePage classes={{}} intl={intl} />);

    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const wrapper = shallowWithIntl(<HomePage classes={{}} intl={intl} />);

    expect(wrapper).toMatchSnapshot();
  });
});
