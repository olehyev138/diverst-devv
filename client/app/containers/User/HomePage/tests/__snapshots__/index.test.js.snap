// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Should render and match the snapshot 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={3}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={true}
    >
      <h1>
        <FormattedMessage
          id="diverst.containers.HomePage.events"
          values={Object {}}
        />
      </h1>
      <Connect(Component) />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs="auto"
    >
      <WithStyles(ForwardRef(Divider))
        orientation="vertical"
      />
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      xs={true}
    >
      <h1>
        <FormattedMessage
          id="diverst.containers.HomePage.news"
          values={Object {}}
        />
      </h1>
      <Connect(Component) />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</Fragment>
`;
